.eslintrc.js

```diff
@@ -73,8 +73,16 @@ module.exports = {
     quotes: ['warn', 'single', { avoidEscape: true }],
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
-
-    // allow debugger during development only
+    indent: ["error", 4],
+    'max-len': ["warn", { "code": 120, "tabWidth": 2 }],
+    'no-trailing-spaces' : ["error", { "skipBlankLines": true }],
+    eqeqeq: ["error", "always"],
+    semi: ["error", "always"],
+    'eol-last': ["error", "never"],
+    "vue/component-tags-order": ["error", {
+      "order": [ [ "script", "template" ], "style" ]
+    }],
```

Comment:
single/double should be consistent for the rules, eg. `'eol-last'` vs `"vue/component-tags-order"` (which I guess will take care of itself when a quotes rules is added haha)

Reply:
Somehow it didn't !!
So I fixed it personally for you (5b303e0c8e18f08103083a1578d4b9768750504e)